Makefile

```diff
@@ -5,7 +5,7 @@ openapi-generator-cli:=java -jar $(openapi-generator-jar)
 
 generator:=java
 library:=okhttp-gson
-services:=payments
+services:=payments checkout
```

Comment:
Do we want to add the checkout or specific service in the `makefile` ?

Reply:
I don't see why not. It does not influence this release, and we'll be using it going forwards